### ISSUE TITLE
`throw-new-error` is already covered by `new-for-builtins`

### ISSUE THREAD

**Issue by fregante** [sindresorhus/eslint-plugin-unicorn#1761]
Should `throw-new-error` be dropped?

```js
throw Error('Soup for one')
```

```
Use `new` when throwing an error.eslint[unicorn/throw-new-error]
Use `new Error()` instead of `Error()`.eslint[unicorn/new-for-builtins]
```

**Comment by fisker** [comment]
https://github.com/sindresorhus/eslint-plugin-unicorn/blob/dffcea8518f6af11a6c712426875d349c2c545d3/rules/throw-new-error.js#L10

**Comment by fregante** [comment]
Isn't that also already covered by ESLint’s own [no-cap](https://eslint.org/docs/rules/new-cap)? 😅

**Comment by fisker** [comment]
I think it's fine to overlap, many cases have multiple problems. In this case, it's auto-fixable anyway, why bother?

**Comment by fisker** [comment]
People may also not enable `no-cap`.

**Comment by fregante** [comment]
> why bother?

Performance? Why have rules that fall 100% inside a (ESLint native, even) rule?

> People may also not enable `no-cap`.

I suppose you could use that argument to split any existing rules into smaller pieces, but  I don't see the reason to, especially when the rule isn't opinionated or problematic, like `new-cap`.

> it's auto-fixable anyway

I suppose that's a good reason. 👍
